src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxEquivalenceTests.cs

```diff
@@ -437,5 +437,15 @@ public void TestUpdateInterpolatedString()
             VerifyEquivalent(tree1, tree2, topLevel: true);
             VerifyNotEquivalent(tree1, tree2, topLevel: false);
         }
+
+        [Fact(DisplayName = "https://github.com/dotnet/roslyn/issues/7380")]
```

Comment:
Why `DisplayName` here?  My understanding is issues were always linked with `WorkItem` attributes.  The effect of using this method is that xunit will use this name vs. the method name when displaying this suite.  I don't think that is the intention.  Additionally it will make it more difficult to programitically link suites and bugs.